cli/js/lib.deno.ns.d.ts

```diff
@@ -17,6 +17,68 @@ declare namespace Deno {
     name: string;
   }
 
+  interface Constructor {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    new (...args: any[]): any;
+  }
+
+  export interface Assert {
+    /** Make an assertion, if not `true`, then throw. */
+    (expr: unknown, msg?: string): asserts expr;
+    /** Use this to stub out methods that will throw when invoked. */
+    unimplemented(msg?: string): never;
```

Comment:
This one should be removed. It's not actually an assertion and wouldn't be used in tests but the implementations -- https://github.com/denoland/deno/issues/4052#issuecomment-589254004 would apply. It also applies to `unreachable()` but that one is conceptually more for tests which are already non-portable so it maybe gets a pass.